test/aws/scaleup.yml

```diff
@@ -120,20 +117,3 @@
       --timeout=0s
     with_items: "{{ pre_scaleup_workers_name }}"
 
-  - name: remove existing machinesets
```

Comment:
These tasks should not be removed.

Reply:
Oops. I forget that these are removed by a script whenever I run the playbooks! Will replace them.